src/Coordination/KeeperLogStore.h

```diff
@@ -14,7 +14,7 @@ namespace DB
 class KeeperLogStore : public nuraft::log_store
 {
 public:
-    KeeperLogStore(const std::string & changelogs_path, uint64_t rotate_interval_, bool force_sync_, bool compress_logs_);
+    KeeperLogStore(const std::string & changelogs_path, LogFileSettings log_file_settings);
```

Comment:
This has broken the debug build:

https://s3.amazonaws.com/clickhouse-builds/45867/ae977b679c2c6fbb1977f920956abc5e17806cf1/binary_tidy/build_log.log

```
Feb 01 11:17:40 /build/utils/keeper-data-dumper/main.cpp:72:24: error: no matching constructor for initialization of 'DB::KeeperLogStore' [clang-diagnostic-error]
Feb 01 11:17:40     DB::KeeperLogStore changelog(argv[2], 10000000, true, settings->compress_logs);
Feb 01 11:17:40                        ^         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Feb 01 11:17:40 /build/src/Coordination/KeeperLogStore.h:17:5: note: candidate constructor not viable: requires 2 arguments, but 4 were provided
Feb 01 11:17:40     KeeperLogStore(const std::string & changelogs_path, LogFileSettings log_file_settings);
Feb 01 11:17:40     ^
Feb 01 11:17:40 /build/src/Coordination/KeeperLogStore.h:14:7: note: candidate constructor (the implicit copy constructor) not viable: requires 1 argument, but 4 were provided
Feb 01 11:17:40 class KeeperLogStore : public nuraft::log_store
Feb 01 11:17:40       ^
Feb 01 11:17:40 188668 warnings and 1 error generated.
```

Reply:
Both tidy and debug build fixed in https://github.com/ClickHouse/ClickHouse/pull/45888